src/Microsoft.VisualStudio.LanguageServices.Razor/Editor/DefaultVisualStudioDocumentTrackerFactory.cs

```diff
@@ -19,30 +17,37 @@ namespace Microsoft.VisualStudio.LanguageServices.Razor.Editor
     [ContentType(RazorLanguage.ContentType)]
     [TextViewRole(PredefinedTextViewRoles.Document)]
     [Export(typeof(IWpfTextViewConnectionListener))]
-    [Export(typeof(VisualStudioDocumentTrackerFactory))]
-    internal class DefaultVisualStudioDocumentTrackerFactory : VisualStudioDocumentTrackerFactory, IWpfTextViewConnectionListener
+    internal class DefaultVisualStudioDocumentTrackerFactory : IWpfTextViewConnectionListener
```

Comment:
This should have a different name YO

Reply:
How do you feel about having a different service that manages creation of document trackers?

```C#
DefaultVisualStudioDocumentTrackerFactory
    VisualStudioDocumentTracker Create(ITextBuffer textBuffer)
```